src/modules/Kdump.rb

```diff
@@ -1173,7 +1177,11 @@ def crash_xen_kernel_values
       # modification.
       # The old value (ensuring the Array format) will be returned.
       if @crashkernel_list_ranges
-        return Array(@crashkernel_xen_param_values.dup)
+        if @crashkernel_xen_param_values.is_a? Symbol
+          return @crashkernel_xen_param_values
```

Comment:
Exactly same comment than above. This method is expected to ensure the Array format. We should not break that. And I would say the same suggested fix applies here.